code/modules/mob/inventory.dm

```diff
@@ -291,7 +291,7 @@
 	return !!locate(/obj/item) in held_items
 
 /mob/proc/drop_all_held_items()
-	. = FALSE
+	. = list()
```

Comment:
My other review is unresolved

Reply:
fixed eyewstab componetn and added documentation!!!!!